app/src/main/java/de/danoeh/antennapod/activity/PreferenceActivity.java

```diff
@@ -128,7 +131,17 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     @Override
     public void onSearchResultClicked(SearchPreferenceResult result) {
-        PreferenceFragmentCompat fragment = openScreen(result.getResourceFile());
-        result.highlight(fragment);
+        int screen = result.getResourceFile();
+        if (screen == R.xml.feed_settings) {
+            AlertDialog.Builder builder = new AlertDialog.Builder(this);
+            builder.setTitle(R.string.feed_settings_label);
+            builder.setMessage(R.string.pref_feed_settings_dialog_msg);
+            builder.setPositiveButton(R.string.pref_ok_label, (dialogInterface, i) -> {
+            });
```

Comment:
If you just pass `null` as the second parameter, it uses a bit less code :)

Please use `android.R.string.ok`, so it does not need to be translated again.

Reply:
Ok, i'll add the changes soon.